backend/src/main/java/hu/flowacademy/meteo/dto/MeasurmentDto.java

```diff
@@ -0,0 +1,40 @@
+package hu.flowacademy.meteo.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import hu.flowacademy.meteo.model.*;
+import hu.flowacademy.meteo.model.enumPackage.Type;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class MeasurmentDto {
+
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private Date date;
+    private Type type;
+
+    private AirData airData;
+    private MiscData miscData;
+    private SoilData soilData;
+    private BatteryData batteryData;
+    private WindData windData;
+
+    public static MeasurmentDto toDto(Measurment measurment) {
```

Comment:
Ezt majd erdemes lesz atvinni a service-be, ahol szukseg lesz ra, ott ugyis elerheto a service, igy staticnak sem kell lennie(Raer a kovektezo story soran implementalni).  Tipp: csinalj egy method overload-ot, ami Collectiont var, es minden elemre meghivja ezt, igy kesobb olvashatobban lehet hasznalni, amikor tobb DTO-t konvertalsz.